frontend/homework/11/src/component/Main.scss

```diff
@@ -0,0 +1,58 @@
+.main-page {
+  display: flex;
+  justify-content: space-between;
+}
+
+.center {
+  margin-left: 128px;
+  margin-right: 128px;
+}
+
+.left {
+  width: 40%;
+}
+
+.right {
+  width: 40%;
+}
+.new {
```

Comment:
1. Use logical naming conventions. Follow BEM naming convention for css/scss
2. Follow nested hierarchy for child elements